Formula/p/pushpin.rb

```diff
@@ -20,16 +21,13 @@ class Pushpin < Formula
   depends_on "pkg-config" => :build
   depends_on "rust" => :build
 
+  depends_on "condure"
```

Comment:
As of a few versions ago, the condure project was folded into pushpin, so this line is not needed and it has no effect.

Reply:
What do you mean by folded? They have "vendored" (e.g. copied the condure code into the pushpin repo?): if this is the case we prefer to use our own condure.

Reply:
Not vendored; ongoing development of the project itself was moved into pushpin.

Basically, we (pushpin maintainers) originally developed & packaged condure separately, but the reason for doing that was mostly historical and not providing much benefit, and so the projects were merged.

We can probably deprecate the condure package. It is not receiving updates and we are not aware of anyone using it other than as a dependency of older pushpins.